apis/Google.Cloud.PubSub.V1/docs/history.md

```diff
@@ -0,0 +1,25 @@
+# Version history
+
+# Version 1.1.0, released 2019-12-10
+
+- [Commit 5742d91](https://github.com/googleapis/google-cloud-dotnet/commit/5742d91): Adds ReceivedMessage.DeliveryAttempt
+- [Commit 1784804](https://github.com/googleapis/google-cloud-dotnet/commit/1784804): Adds dead letter policy
+- [Commit 50658e2](https://github.com/googleapis/google-cloud-dotnet/commit/50658e2): Add format method for all resource name types
+- [Commit c3af927](https://github.com/googleapis/google-cloud-dotnet/commit/c3af927): Fix error in handling non-ordered batch errors ([issue 3133](https://github.com/googleapis/google-cloud-dotnet/issues/3133))
+- [Commit 11d9105](https://github.com/googleapis/google-cloud-dotnet/commit/11d9105): Disable the ordering-keys feature in publisher ([issue 3122](https://github.com/googleapis/google-cloud-dotnet/issues/3122))
+  - This feature is not yet available on the live server. This PR will be reverted once it is available.
```

Comment:
Maybe remove this and the one that added the feature two bullets down. It's weird as version history.
Unless the feature is now actually live?

Reply:
Will ask Chris.

Reply:
Okay, have spoken to Chris, and pushed a new version. I think it's worth acknowledging that the code is present, but not enabled yet.